.github/workflows/code-pushup.yml

```diff
@@ -21,8 +21,8 @@ jobs:
     env:
       CP_SERVER: ${{ secrets.CP_SERVER }}
       CP_API_KEY: ${{ secrets.CP_API_KEY }}
-      CP_ORGANIZATION: code-pushup
-      CP_PROJECT: cli
+      CP_ORGANIZATION: push-based
+      CP_PROJECT: nx-verdaccio
```

Comment:
I created a new project [`push-based/nx-verdaccio`](https://staging.code-pushup.dev/portal/push-based/nx-verdaccio/branch/main/dashboard) in the portal, but there's a few steps missing to make uploads work:
- create an API key (or reuse existing) in `push-based` organization and update the `CP_API_KEY` secret
- add `upload` configuration to `code-pushup.config.ts`
- install `@code-pushup/portal-client` package